core/src/main/java/hudson/util/FormApply.java

```diff
@@ -66,12 +68,16 @@ public static boolean isApply(StaplerRequest req) {
         return Boolean.parseBoolean(req.getParameter("core:apply"));
     }
 
+
     /**
      * Generates the response for the asynchronous background form submission (AKA the Apply button.)
      * <p>
      * When the response HTML includes a JavaScript function in a pre-determined name, that function gets executed.
      * This method generates such a response from JavaScript text.
+     *
+     * @deprecated use {#{@link #showNotification(String, NotificationType)}} instead, which is CSP compatible version
```

Comment:
```suggestion
     * @deprecated use {@link #showNotification(String, NotificationType)} instead, which is CSP compatible version
```
Otherwise, it'll render as `use {#showNotification(String, FormApply.NotificationType)}...` instead of `use showNotification(String, FormApply.NotificationType)...`

Reply:
Thanks for the catch, addressed in https://github.com/jenkinsci/jenkins/pull/8351/commits/e8716aee22a04b475d0c6a499b186ca2f01d6c75